thirdparty/vars.sh

```diff
@@ -583,6 +595,8 @@ export TP_ARCHIVES=(
     'JSONCPP'
     'LIBUUID'
     'ALI_SDK'
+    'WASMTIME'
+    'WASMTIME_CPP'
```

Comment:
Hi @taptao , it seems that these libraries don't support macOS (arm64). I think you would better remove these lines here and add them somewhere else after checking the platform. Currently, this script can not be ran successfully on macOS (arm64).

Reply:
> I noticed this, but my env is mac arm, and the compilation was successful after installing rust, did you get any errors?

Hi @taptao , thanks for your prompt reply. I ran the script successfully after setting Rust up.